src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1036,7 +1036,7 @@
     },
     {
       "run": "after",
-      "snippetPath": "publication-period/publication-period-functions.sql",
+      "snippetPath": "publication-period/drop-publication-period-functions.sql",
       "fromModuleVersion": "23.0.0"
     },
```

Comment:
You should remove this entry and create a new one with current module version in the end of the file.

Reply:
done